README.md

```diff
@@ -455,10 +455,10 @@ Because different clients have different processes of exiting please make sure t
 Make sure your validator node is running before starting the exit command.
 
 ```sh
-# Exit validator keys with Prysm for Mainnet
+# Exit validator keys with Prysm/Teku for Mainnet
```

Comment:
Please extend `validator exit` descriptions for all consensus clients (how it works and if it affects all keys, or you need to select - you can explain what will happen when you execute `sudo lukso validator exit` - about external consensus client CLI